### PR TITLE
change page title

### PR DESCRIPTION
**Pull request by brnv**

### CODE REVIEW

**Comment by kovetskiy** [comment]
I'm not sure why we would need it. If we are going for DaaC (Documentation
as a Code) we don't want to store some external ID in source documentation.

**Comment by brnv** [comment]
up

File: main.go [hunk 1]
```diff
@@ -285,7 +285,40 @@ func main() {
 
 	var target *PageInfo
 
-	if meta != nil {
+	var rename *PageInfo
+
+	if len(meta.Title) > 1 {
+		page, _ := api.findPage(meta.Space, meta.Title[len(meta.Title)-1])
+		if page == nil {
+			for _, title := range meta.Title {
```
Comment (kovetskiy) [inline main.go]: we need to go in reversed order

Comment (brnv) [reply main.go]: check pls

**Pull request merged**